### PR TITLE
Fixed some spelling and grammar issues

### PR DESCRIPTION
**Pull request by FormalSnake**
Also fixed some of the capital letters in random spots.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,13 +36,13 @@
...
-<li><p>(Option 2) You could edit the file --> <code>chrome</code> --> <code>components</code> --> <code>ogx_root-personal.css</code> line <code>18</code><p></li></ul></br>
+   <li><p>(Option 1) In the directory: <code>chrome</code> >> <code>images</code> >> <code>newtab</code> there is a set of wallpapers, the image that the theme will select by default is the one with the name: "wallpaper.png", you could rename the image to something else, but you will need to change some variables. If you don't want a wallpaper just delete the file with the specified name. You can find more images here: <a href="https://imgur.com/a/q8SYEUJ"><b> >>This Gallery<< </b></a>. <b>Restarting the browser after changing the wallpaper is required. </b></p></li>
```
Comment (Godiesc) [inline README.md:41]: I'm not sure if your change in this part is what I tried to said. after the part: "wallpaper.png".

Comment (FormalSnake) [reply README.md:41]: It is what you tried to say, it just was not that clear.

Comment (FormalSnake) [reply README.md:41]: luckily i speak spanish.

Comment (Godiesc) [reply README.md:41]: (ES) you could rename the image you want with that name and the theme will select that one, you could add your own images.
(EN) you could rename the image to something else, but you will need to change some variables.


In this part I don't think is the same idea.

Comment (FormalSnake) [reply README.md:41]: ill change that, one moment

**Pull request closed without merging**